en/Advanced topics/Catalyst license.md

```diff
@@ -1,13 +1,13 @@
-Catalyst license is an optional license to use Obsidian under. It has three tiers ($25, $50, $100), all of them give you [[Insider builds|access to insider builds]].
+Obsidian is free for personal use. A Catalyst license is an optional license that provides the following benefits:
 
-Please note that Catalyst is optional and ==does not give you any additional feature==.
+1. It gives you early access to the latest, cutting edge versions of the app
```

Comment:
Would appreciate some punctuation at the end of each list item, but not sure if comma, period, or semicolon works the best.

Reply:
Np. I'll add semicolons.